{
  "document": {
    "aggregate_severity": {
      "namespace": "https://www.suse.com/support/security/rating/",
      "text": "moderate"
    },
    "category": "csaf_vex",
    "csaf_version": "2.0",
    "distribution": {
      "text": "Copyright 2023 SUSE LLC. All rights reserved.",
      "tlp": {
        "label": "WHITE",
        "url": "https://www.first.org/tlp/"
      }
    },
    "lang": "en",
    "notes": [
      {
        "category": "summary",
        "text": "SUSE CVE-2015-2317",
        "title": "Title"
      },
      {
        "category": "description",
        "text": "The utils.http.is_safe_url function in Django before 1.4.20, 1.5.x, 1.6.x before 1.6.11, 1.7.x before 1.7.7, and 1.8.x before 1.8c1 does not properly validate URLs, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a control character in a URL, as demonstrated by a \\x08javascript: URL.",
        "title": "Description of the CVE"
      },
      {
        "category": "legal_disclaimer",
        "text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
        "title": "Terms of use"
      }
    ],
    "publisher": {
      "category": "vendor",
      "contact_details": "https://www.suse.com/support/security/contact/",
      "name": "SUSE Product Security Team",
      "namespace": "https://www.suse.com/"
    },
    "references": [
      {
        "category": "external",
        "summary": "CVE-2015-2317",
        "url": "https://www.suse.com/security/cve/CVE-2015-2317"
      },
      {
        "category": "external",
        "summary": "SUSE Security Ratings",
        "url": "https://www.suse.com/support/security/rating/"
      },
      {
        "category": "external",
        "summary": "SUSE Bug 923176 for CVE-2015-2317",
        "url": "https://bugzilla.suse.com/923176"
      },
      {
        "category": "external",
        "summary": "Advisory link for SUSE-SU-2015:0694-1",
        "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-April/001339.html"
      },
      {
        "category": "external",
        "summary": "Advisory link for SUSE-SU-2015:0695-1",
        "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-April/001340.html"
      },
      {
        "category": "external",
        "summary": "Advisory link for SUSE-SU-2015:1109-1",
        "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-June/001452.html"
      },
      {
        "category": "external",
        "summary": "Advisory link for SUSE-SU-2015:1112-1",
        "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-June/001453.html"
      },
      {
        "category": "external",
        "summary": "Advisory link for openSUSE-SU-2015:0643-1",
        "url": "https://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html"
      },
      {
        "category": "external",
        "summary": "Advisory link for openSUSE-SU-2015:1598-1",
        "url": "https://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html"
      }
    ],
    "title": "SUSE CVE CVE-2015-2317",
    "tracking": {
      "current_release_date": "2023-02-15T05:20:55Z",
      "generator": {
        "date": "2023-02-15T05:20:55Z",
        "engine": {
          "name": "cve-database.git:bin/generate-csaf-vex.pl",
          "version": "1"
        }
      },
      "id": "CVE-2015-2317",
      "initial_release_date": "2023-02-15T05:20:55Z",
      "revision_history": [
        {
          "date": "2023-02-15T05:20:55Z",
          "number": "2",
          "summary": "Current version"
        }
      ],
      "status": "interim",
      "version": "2"
    }
  },
  "product_tree": {
    "branches": [
      {
        "branches": [
          {
            "branches": [
              {
                "category": "product_name",
                "name": "SUSE Enterprise Storage 1.0",
                "product": {
                  "name": "SUSE Enterprise Storage 1.0",
                  "product_id": "SUSE Enterprise Storage 1.0",
                  "product_identification_helper": {
                    "cpe": "cpe:/o:suse:ses:1.0"
                  }
                }
              },
              {
                "category": "product_name",
                "name": "SUSE OpenStack Cloud 4",
                "product": {
                  "name": "SUSE OpenStack Cloud 4",
                  "product_id": "SUSE OpenStack Cloud 4",
                  "product_identification_helper": {
                    "cpe": "cpe:/o:suse:cloud:4"
                  }
                }
              },
              {
                "category": "product_name",
                "name": "SUSE OpenStack Cloud 5",
                "product": {
                  "name": "SUSE OpenStack Cloud 5",
                  "product_id": "SUSE OpenStack Cloud 5",
                  "product_identification_helper": {
                    "cpe": "cpe:/o:suse:cloud:5"
                  }
                }
              },
              {
                "category": "product_version",
                "name": "python-Django-1.6.11-0.7.1",
                "product": {
                  "name": "python-Django-1.6.11-0.7.1",
                  "product_id": "python-Django-1.6.11-0.7.1",
                  "product_identification_helper": {
                    "cpe": "cpe:2.3:a:djangoproject:django:1.6.11:*:*:*:*:*:*:*"
                  }
                }
              },
              {
                "category": "product_version",
                "name": "python-Django-1.6.11-4.1",
                "product": {
                  "name": "python-Django-1.6.11-4.1",
                  "product_id": "python-Django-1.6.11-4.1",
                  "product_identification_helper": {
                    "cpe": "cpe:2.3:a:djangoproject:django:1.6.11:*:*:*:*:*:*:*"
                  }
                }
              },
              {
                "category": "product_version",
                "name": "python-django-1.5.12-0.9.1",
                "product": {
                  "name": "python-django-1.5.12-0.9.1",
                  "product_id": "python-django-1.5.12-0.9.1",
                  "product_identification_helper": {
                    "cpe": "cpe:2.3:a:djangoproject:django:1.5.12:*:*:*:*:*:*:*"
                  }
                }
              }
            ],
            "category": "product_family",
            "name": "SUSE Linux Enterprise"
          }
        ],
        "category": "vendor",
        "name": "SUSE"
      }
    ],
    "relationships": [
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python-Django-1.6.11-4.1 as component of SUSE Enterprise Storage 1.0",
          "product_id": "SUSE Enterprise Storage 1.0:python-Django-1.6.11-4.1"
        },
        "product_reference": "python-Django-1.6.11-4.1",
        "relates_to_product_reference": "SUSE Enterprise Storage 1.0"
      },
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python-django-1.5.12-0.9.1 as component of SUSE OpenStack Cloud 4",
          "product_id": "SUSE OpenStack Cloud 4:python-django-1.5.12-0.9.1"
        },
        "product_reference": "python-django-1.5.12-0.9.1",
        "relates_to_product_reference": "SUSE OpenStack Cloud 4"
      },
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python-Django-1.6.11-0.7.1 as component of SUSE OpenStack Cloud 5",
          "product_id": "SUSE OpenStack Cloud 5:python-Django-1.6.11-0.7.1"
        },
        "product_reference": "python-Django-1.6.11-0.7.1",
        "relates_to_product_reference": "SUSE OpenStack Cloud 5"
      }
    ]
  },
  "vulnerabilities": [
    {
      "cve": "CVE-2015-2317",
      "ids": [
        {
          "system_name": "SUSE CVE Page",
          "text": "https://www.suse.com/security/cve/CVE-2015-2317"
        }
      ],
      "notes": [
        {
          "category": "general",
          "text": "The utils.http.is_safe_url function in Django before 1.4.20, 1.5.x, 1.6.x before 1.6.11, 1.7.x before 1.7.7, and 1.8.x before 1.8c1 does not properly validate URLs, which allows remote attackers to conduct cross-site scripting (XSS) attacks via a control character in a URL, as demonstrated by a \\x08javascript: URL.",
          "title": "Vulnerability description"
        }
      ],
      "product_status": {
        "fixed": [
          "SUSE Enterprise Storage 1.0:python-Django-1.6.11-4.1",
          "SUSE OpenStack Cloud 4:python-django-1.5.12-0.9.1",
          "SUSE OpenStack Cloud 5:python-Django-1.6.11-0.7.1"
        ]
      },
      "references": [
        {
          "category": "external",
          "summary": "CVE-2015-2317",
          "url": "https://www.suse.com/security/cve/CVE-2015-2317"
        },
        {
          "category": "external",
          "summary": "SUSE Security Ratings",
          "url": "https://www.suse.com/support/security/rating/"
        },
        {
          "category": "external",
          "summary": "SUSE Bug 923176 for CVE-2015-2317",
          "url": "https://bugzilla.suse.com/923176"
        },
        {
          "category": "external",
          "summary": "Advisory link for SUSE-SU-2015:0694-1",
          "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-April/001339.html"
        },
        {
          "category": "external",
          "summary": "Advisory link for SUSE-SU-2015:0695-1",
          "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-April/001340.html"
        },
        {
          "category": "external",
          "summary": "Advisory link for SUSE-SU-2015:1109-1",
          "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-June/001452.html"
        },
        {
          "category": "external",
          "summary": "Advisory link for SUSE-SU-2015:1112-1",
          "url": "https://lists.suse.com/pipermail/sle-security-updates/2015-June/001453.html"
        },
        {
          "category": "external",
          "summary": "Advisory link for openSUSE-SU-2015:0643-1",
          "url": "https://lists.opensuse.org/opensuse-updates/2015-04/msg00001.html"
        },
        {
          "category": "external",
          "summary": "Advisory link for openSUSE-SU-2015:1598-1",
          "url": "https://lists.opensuse.org/opensuse-updates/2015-09/msg00035.html"
        }
      ],
      "remediations": [
        {
          "category": "vendor_fix",
          "details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
          "product_ids": [
            "SUSE Enterprise Storage 1.0:python-Django-1.6.11-4.1",
            "SUSE OpenStack Cloud 4:python-django-1.5.12-0.9.1",
            "SUSE OpenStack Cloud 5:python-Django-1.6.11-0.7.1"
          ]
        }
      ],
      "threats": [
        {
          "category": "impact",
          "date": "2015-03-19T09:41:59Z",
          "details": "moderate"
        }
      ],
      "title": "CVE-2015-2317"
    }
  ]
}
